omero/developers/Python.rst

```diff
@@ -310,6 +341,31 @@ Read data
                 well.getImage(index).getName(),\
                 well.getImage(index).getId()
 
+-  **List all annotations on an object. Get text from tags**
```

Comment:
the text I am getting here, is that the same thing as, "lowly said", the "name" of a tag ?

Reply:
I don't want to use 'name' because Tag doesn't have a 'name' attribute but other objects do.  I'll use textValue, which is the name of the attribute.